File: Solutions/2021/1/a.js
// December 1st, 2021
const fs = require("fs");

const list = fs.readFileSync(`${__dirname}/input.txt`, "utf8").split('\r\n').map(x => parseInt(x));
let count = 0;
for (var i = 1; i < list.length; i++) {
	if (list[i] > list[i - 1]) count++;
}
console.log(count);

File: Solutions/2021/1/a.py
# December 25th, 2021

file = open(f"{__file__}/../input.txt", "r")
lines = file.read().split("\n")

count = 0
for i in range(1, len(lines) - 1):
	if int(lines[i]) > int(lines[i - 1]):
		count += 1
print(count)
